### DISCUSSION TITLE
请问博客中流程图使用到的是什么插件？

### DISCUSSION THREAD

**Discussion by zzcyeah** [mebtte/animal-photosynthesis#16] [Q & A] [answered]
这个手绘体就很好看
![image](https://user-images.githubusercontent.com/34113572/167536607-46a662c5-0cc6-4b77-859c-70bb74db2b7c.png)

**Comment by mebtte** [comment, marked as answer]
@zzcyeah 支持中文手写字体的 Excalidraw
https://github.com/mebtte/excalidraw

**Comment by zzcyeah** [comment]
这是来自QQ邮箱的假期自动回复邮件。您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。
